.vib/consul/goss/goss.yaml

```diff
@@ -0,0 +1,12 @@
+gossfile:
+  # Goss tests exclusive to the current container
+  ../../consul/goss/consul.yaml: {}
```

Comment:
Similarly to https://github.com/bitnami/containers/pull/30098, I don't see any added value having a `binary --help` test when there is already `binary --version` one. It doesn't hurt, but I'm in favour of not having it if is not valuable. In this case, we'll be able to remove the `consul.yalm` as well.

Besides that, the PR LGTM; I'll wait for the VIB trigger removal.